typescript-operators-and-expressions/main.ts

```diff
@@ -0,0 +1,38 @@
+const width: number = 10;
+const height: number = 20;
+const area: number = width * height;
+console.log('the value of the area is:', area);
+console.log('the typeof area is:', typeof area);
+
+const bill: number = 44;
+const payment: number = 100;
+const change: number = payment - bill;
+console.log('the value of change:', change);
+console.log('the typeof change:', typeof change);
+
+const quizzes: number = 55;
+const midterm: number = 77;
+const final: number = 99;
+const grade: number = (quizzes + midterm + final) / 3;
+console.log('the value of grade:', grade);
+console.log('the typeof grade:', typeof grade);
+
+const firstName: string = 'Carolyne';
+const lastName: string = 'Golias';
+const fullName: string = firstName + lastName;
```

Comment:
Don't forget to add a space! There should be a space between your first and last name